### PR TITLE
style: Introduce editorconfig

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
EditorConfig[1] is a way to create a common style across a repository that's not bound to a single language. It's a static analysis and formatting guide that could help us adopt one style in this repository. As such I've introduced a very small change in this first iteration to get started and see if it's for us.

[1]: https://editorconfig.org/

### CODE REVIEW

**Review by Pezmc — APPROVED** [review]
I'm in strong support of adding this to all FlowForge repo's, but given it's not built into a lot of editors, I'd suggest adding a reference to it in the developer docs

**Comment by ZJvandeWeg** [comment]
> I'm in strong support of adding this to all FlowForge repo's, but given it's not built into a lot of editors, I'd suggest adding a reference to it in the developer docs

✅

File: src/handbook/development/contributing.md [hunk 1]
```diff
@@ -16,10 +17,18 @@ If you're using VSCode, then we recommend using the [ESLint extenstion](https://
...
-In the case of working with `vue` or `njk` files (found in the [frontend](https://github.com/flowforge/flowforge/tree/main/frontend) and [website](https://github.com/flowforge/website) repositories), then you can add `vue` and `njk` to the `Eslint: Probe` setting in order to enable auto-formatting on save for these file types.
+In the case of working with `vue` or `njk` files (found in the [frontend](https://github.com/flowforge/flowforge/tree/main/frontend) and [website][website-repo] repositories), then you can add `vue` and `njk` to the `Eslint: Probe` setting in order to enable auto-formatting on save for these file types.
 
 <img width="478" alt="ESLint - Probe" src="../images/eslint_probe.png">
 
+#### Editor Config
+
+The [website repository][website-repo] uses a [`.editorconfig`](https://editorconfig.org)
+to allow editors to automatically pick up the correct style for that repository.
+The best editor, neovim, has this pre-installed. If you're using VSCode, an
```
Comment (Pezmc) [inline src/handbook/development/contributing.md:28]: Let's bike shed on this for the next 1-2 weeks!

Comment (ZJvandeWeg) [reply src/handbook/development/contributing.md:28]: Really hoped a line in the middle would do the trick 😄 

```suggestion
Some editors, like neovim, has this pre-installed. If you're using VSCode, an
```

**Pull request merged**